src/views/blocks/ImageTextBlock.php

```diff
@@ -4,27 +4,34 @@
 /**
  * @var $this \luya\cms\base\PhpBlockView
  */
+
+$image = $this->extraValue('image');
+$text = $this->extraValue('text');
 ?>
-<?php if ($this->extraValue('image') && $this->extraValue('text')): $imageCaption = $this->extraValue('image')['caption'];?>
-    <div>
-        <?= Html::img($this->extraValue('image')['source'], [
-            'class' => ($this->varValue('imagePosition', 'left') == 'left') ? 'pull-left img-responsive' : 'pull-right img-responsive',
-            'alt' => $imageCaption === null ? '' : $imageCaption,
-            'title' => $imageCaption,
-            'width' => $this->cfgValue('width', null),
-            'height' => $this->cfgValue('height', null),
-            'style' => (($this->varValue('imagePosition', 'left') == 'left') ? "margin-right:{$this->cfgValue('margin', '20px')}" : "margin-left:{$this->cfgValue('margin', '20px')}") . $this->cfgValue('margin', '20px', ';margin-bottom:{{margin}};max-width:50%;'),
-        ])?>
-        <div>
-            <?= $this->extraValue('text'); ?>
+<?php if ($image && $text): ?>
```

Comment:
I dont prefer this compact variant because it is often harder to read and the it doing too many things in one line. For me it is much clearer when a variable is defined in it own line.

The reason to create the $image and $text variables is that the `extraValue` is called multiple time and it is easier too read `if ($image && $text) :` than `if ($this->extraValue('image') && $this->extraValue('text'))`

Reply:
@boehsermoe Okay, keep it like this if you like it ;-) I personal don't like it if things like this `    <?php $imageCaption = $image['caption']; ?>` are just part of html notation with tabs and in general we try to avoid this, over all elements in LUYA, but ok. (I would like to micro test this part of code, but regarding ram usage i try to avoid create variables if its not need, only for readability, as it consumes memory)